ios/Podfile

```diff
@@ -24,5 +25,28 @@ target 'SampleApp' do
 end
 
 target 'Notification Service' do
-  pod 'CustomerIOMessagingPushAPN', :podspec => 'https://raw.githubusercontent.com/customerio/customerio-ios/1.1.1/CustomerIOMessagingPushAPN.podspec'
+  pod 'CustomerIO/Tracking', '~> 1.1'
```

Comment:
Same for Notification Service. https://github.com/customerio/amiapp-reactnative/pull/10/files#r923136545